src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -177,10 +392,28 @@ public static ElementTag createOption(String text, String value) {
         return result;
     }
     
+    /**
+     * Returns the nationalities as HTML code.
+     * @param existingNationality
+     * @return
```

Comment:
Avoid empty javadoc tags. If you don't have anything else to say about that @param or @return, then omit the entire line.